src/CustomElementInternals.js

```diff
@@ -42,6 +42,20 @@ export default class CustomElementInternals {
   }
 
   /**
+   * @param {!Node} node
+   * @return {!CustomElementDefinition|null}
+   */
+  nodeToDefinition(node) {
```

Comment:
Getting the CE definition for a DOM element used to be as straightforward as looking things up by `localName`. But now it's a bit more complicated since you have to deal with customized builtins where the localName is not the same as the name.